src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml

```diff
@@ -19,18 +19,11 @@ the MIT License. See LICENSE in the project root for license information. -->
 
         <Style TargetType="Button">
             <Setter Property="BorderBrush" Value="{StaticResource SystemBaseLowColor}"/>
-            <Setter Property="Height" Value="30" />
-            <Setter Property="Width" Value="100" />

```

Comment:
I'd say keep these two values here since the all of the color picker buttons use these two values, and for the new `Add` and `Delete` buttons, explicitly give them `Height="auto", Width="auto"` to override?

Or, you can give this Style an `x:Key="ColorPickerButtonStyle`, and apply it to each of the color picker buttons, either way sounds gud.